docs/markdown/Getting Help/the-pants-community/maintainers.md

```diff
@@ -4,41 +4,84 @@ slug: "maintainers"
+
+We expect Maintainers to be able to answer questions about their areas of expertise as they are called upon. If you are unable to respond to a question in a timely manner, we'd appreciate it if you could make that clear, so someone else can offer help, or a decision can be made in your absence.
+
+If there are demands on your time that prevent you from participating in the Pants community, that's OK! We can usually cope with this, especially if you can let other maintainers know in advance. That way, we can prioritize work that you might have otherwise taken on, and ensure there isn't a daunting pile of issues and pull requests awaiting you when you return.
 
-Maintainer candidates are nominated by existing maintainers from among the wider contributor base. Criteria for nomination include:
+If you no longer feel that you are able to contribute effectively as a maintainer, you may request to be converted to a [Maintainer Emeritus](doc:the-pants-community#former-team-members).
 
-- The candidate's desire to become a maintainer.
-- A track record of good contributions.
-- A friendly, helpful, positive attitude.
 
-If a contributor has been nominated, and is willing to become a maintainer, then their candidacy will be discussed and voted on by the existing maintainers.
+### Release duty is optional
 
-Maintainer onboarding
+Some maintainers also accept responsibility for publishing both development and stable releases of Pants. Releases are usually published once per week.
+
+We use a [Google Calendar](https://calendar.google.com/calendar/b/0/embed?src=hvd8qnf6fnp5klnk7u46q1noeo@group.calendar.google.com&ctz=America/Los_Angeles) to coordinate who is on release duty in a given week. When it is time for a release, the Maintainer who has release duty that week is responsible for updating the release documentation, creating release builds, and stewarding them through the review and [release process](doc:release-process).
+
+Release duty is a commitment beyond that of a Maintainer, and we do not expect Maintainers to take on this additional responsibility without enthusiastically opting in.
+
+
+Becoming a Maintainer
 ---------------------
 
+Maintainer candidates are nominated by existing Maintainers from among the wider contributor base. Criteria for nomination include:
+
+-   Candidate has a record of good contributions to Pants
+-   Candidate has a record of collaboration with Pants community members
+-   Candidate understands the principles described on this page
+-   Candidate is committed to upholding Pants community standards, including the Code of Conduct
+
+Candidates will only be publicly nominated after one Maintainer has determined they are willing to become a Maintainer. 
```

Comment:
The nomination occurs in private, so there isn't really a public nomination, per-se?